.github/workflows/download_upload_cycle.yml

```diff
@@ -0,0 +1,43 @@
+name: download_upload_cycle
+on:
+  schedule:
```

Comment:
I'd recommend adding `pull_request:` to the `on` conditions so this gets run once on pull requests.  We might also want to add `push:` to this and the test.yml workflow's conditions, so we get tests on pushed branches before they are made into PRs.

Reply:
Changed in [7409ccb](https://github.com/girder/shapeworks-cloud/pull/291/commits/7409ccb8e6cb3a879587b2b8f5417ad5889b712c)

Reply:
I'm noticing that the tests all run twice if we have both `pull_request` and `push`. Should I change these to just push?